### DISCUSSION TITLE
Include dynamic link to unreleased features in CHANGELOG header

### DISCUSSION THREAD

**Discussion by anishkny** [absolute-version/commit-and-tag-version#45] [Q&A] [answered]
Hello,

First up thanks for forking `standard-version` and shepherding it on - cheers! 🎉 

I have been trying to figure this out but can't. 

I would like to include something like the following in my generated `CHANGELOG` header:

> For unreleased features, see: `{{host}}/{{owner}}/{{repository}}/compare/{{currentTag}}...main`

So for example, in my project [integrify](https://github.com/anishkny/integrify), the link would point to: https://github.com/anishkny/integrify/compare/v4.1.1...main . This lets you see a list of commits landed on main but not released yet.

I need the tags interpolated in the changelog header. Is this possible?

**Comment by TimothyJones** [comment, marked as answer]
Apologies for not seeing this! Looks like I answered it over [here](https://github.com/absolute-version/commit-and-tag-version/issues/58)
